pkg/stanza/operator/input/windows/xml.go

```diff
@@ -31,6 +32,7 @@ type EventXML struct {
 	Security         *Security        `xml:"System>Security"`
 	Execution        *Execution       `xml:"System>Execution"`
 	EventData        []EventDataEntry `xml:"EventData>Data"`
+	UserData         *AnyXML          `xml:"UserData"`
```

Comment:
I think here we want something like:
```suggestion
	UserData         *UserData          `xml:"UserData"`
```

and `UserData` to be something like:
```golang
	Elements	[]AnyXML
	Attributes	map[string]string
```

This is matches the event schema.